tools/llm_bench/task/text_generation.py

```diff
@@ -227,7 +227,6 @@ def run_text_generation_genai(input_text, num, model, tokenizer, args, iter_data
     gen_config = model.get_generation_config()
     gen_config.max_new_tokens = max_gen_tokens
     gen_config.num_beams = args["num_beams"]
```

Comment:
please, remove `set_seed(args['seed'])` on str 201 and 356 and add `gen_config.rng_seed= args["seed"]` and to here `run_text_generation_genai_with_stream`

Reply:
Done, thanks!